metron-platform/metron-elasticsearch/src/main/config/elasticsearch.properties

```diff
@@ -57,10 +57,13 @@ org.apache.metron.metrics.TelemetryParserBolt.fails=true
 
 bolt.hdfs.batch.size=5000
 bolt.hdfs.field.delimiter=|
+bolt.hdfs.rotation.policy=org.apache.storm.hdfs.bolt.rotation.TimedRotationPolicy
+bolt.hdfs.rotation.policy.units=DAYS
+bolt.hdfs.rotation.policy.count=1
+
 bolt.hdfs.file.rotation.size.in.mb=5
```

Comment:
Oh, I get it. Yeah, we'll likely need to expose it as configs, but for now it makes sense.